utils/scale-test

```diff
@@ -269,6 +275,8 @@ def main():
 
     args = parser.parse_args(sys.argv[1:])
     (rng, runs) = run_many(args)
+    if args.tmpdir != None:
+        shutil.rmtree(args.tmpdir)
```

Comment:
Is there any reason we have to delete this at all?   If there is, can we do it *before* the test instead of after?  Otherwise we delete valuable artifacts for debugging failures.